src/components/NavigationFolder.vue

```diff
@@ -69,6 +69,15 @@
 				>
 					{{ t('mail', 'Clear locally cached data, in case there are issues with synchronization.') }}
 				</ActionButton>
+
+				<ActionCheckbox
+					v-if="notVirtual"
+					:checked="isSubscribed"
+					:disabled="changeSubscription"
+					@update:checked="subscribeFolder"
+				>
+					{{ t('mail', 'Subscribed') }}
```

Comment:
Should it just be `Subscribe`? Not sure :thinking:

Reply:
The text is next to the checkbox that shows the current status. So I would leave it like this.

Reply:
Fine by me :)